bootstrap_db.py

```diff
@@ -1,13 +1,17 @@
-import os
-
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy import create_engine
 
 from models import Role, Outcome, Base
 
-def bootstrap(db_host='localhost', db_password='docker'):
-    engine = create_engine('postgresql://postgres:%s@%s:5432' % (db_password, db_host))
+def bootstrap():
+    engine = create_engine(
```

Comment:
Can this use `db_session` from the other file?

Reply:
Since this drops and creates table it needs a reference to the engine object which the `db_session` does not provide

Reply:
Drop tables huh ( ͡° ͜ʖ ͡°)